### ISSUE TITLE
Does ScaledYOLOv4 use the same encoding of bounding box coordinates as YOLOv3 

### ISSUE THREAD

**Issue by songyuc** [WongKinYiu/ScaledYOLOv4#90]
Hi, guys,
I want to know whether ScaledYOLOv4 uses the same encoding of bounding box coordinates as YOLOv3, like,
![image](https://user-images.githubusercontent.com/27288110/101898213-b0396380-3be6-11eb-933d-48bc028d3d1f.png)

Any answer or idea will be appreciated!

**Comment by WongKinYiu** [comment]
No, the encoding/decoding method is changed in yolov4.
yolov4-tiny used same encoding/decoding method as yolov4.
the encoding/decoding method is proposed by glenn, which solved the grid sensitivity problem.
glenn further replaced exponential function by sigmoid function to avoid nan problem in training process.
yolov4-csp and yolov4-large models use this new encoding/decoding method.

reference: https://github.com/AlexeyAB/darknet/issues/3293 https://github.com/AlexeyAB/darknet/issues/6987#issuecomment-729218069

**Comment by songyuc** [comment]
Hi, @WongKinYiu 
Today I read the code of ScaledYOLOv4, and translate the decoding as these formulas,
![image](https://user-images.githubusercontent.com/27288110/101907931-39a46200-3bf6-11eb-81eb-7d3bd09016c8.png)
I hope you can give me more correction advices.

**Comment by WongKinYiu** [comment]
yes, the formulas are totally same as https://github.com/AlexeyAB/darknet/issues/6987#issuecomment-729218069

**Comment by songyuc** [comment]
@WongKinYiu , I am not sure about why there is a factor of 2. Is this factor of 2 would help mitigate the issue of sigmoid's gradiet vanishing?

**Comment by saikrishnadas** [comment]
So, Is it possible to use the same yolo annotations(.txt) as YOLOv4 darknet in Scaled-Yolov4?
